### PR TITLE
removed the feed button from the user popup

### PR DESCRIPTION
**Pull request by TheGreenOak**
it looks weird but the whole popup needs to move and be more like an actual popup and less like a modal

### CODE REVIEW

File: cv_next/components/layout/navbar/popup.tsx [hunk 1]
```diff
@@ -13,11 +13,11 @@ const navLinks = [
...
-  {
-    route: "Feed",
-    path: "/feed",
-    req_login: true,
-  },
+  // {
+  //   route: "Feed",
+  //   path: "/feed",
+  //   req_login: true,
+  // },
```
Comment (Aradon267) [inline cv_next/components/layout/navbar/popup.tsx:20]: We can just delete these as we don't want them anymore

Comment (Aradon267) [inline cv_next/components/layout/navbar/popup.tsx:20]: We can just delete these as we don't want them anymore

**Review by Aradon267 — CHANGES_REQUESTED** [review]

File: cv_next/components/layout/navbar/popup.tsx (hunk 1 above)
Comment (Aradon267) [reply cv_next/components/layout/navbar/popup.tsx:20]: @TheGreenOak reminding you this

**Review by YechiamW3 — APPROVED** [review]

**Review by TheGreenOak — COMMENTED** [review]
now its removed instead of commented

**Comment by TheGreenOak** [comment]
@Aradon267 why is the merging still blocked?

**Review by Aradon267 — APPROVED** [review]

**Comment by Aradon267** [comment]
I think that it was because  I also needed to approve this...
Can be merged now!

**Pull request merged**